### DISCUSSION TITLE
How to set mobile metrics on raw driver

### DISCUSSION THREAD

**Discussion by bshsbr** [seleniumbase/SeleniumBase#3300] [Q&A] [answered]
I wonder if it's possible to set --metrics to raw driver just like tests
from seleniumbase import Driver

driver = Driver(mobile=True)
try:
    driver.open("https://www.skype.com/en/get-skype/")
    driver.assert_element('[aria-label="Microsoft"]')
    driver.assert_text("Download Skype", "h1")
    driver.highlight("div.appBannerContent")
    driver.highlight("h1")
    driver.assert_text("Skype for Mobile", "h2")
    driver.highlight("h2")
    driver.highlight("#get-skype-0")
    driver.highlight_click("span[data-dropdown-icon]")
    driver.highlight("#get-skype-0_android-download")
    driver.highlight('[data-bi-id*="ios"]')
finally:
    driver.quit()

pytest test_swag_labs.py --mobile --metrics="360,640,2"

**Comment by mdmintz** [comment, marked as answer]
Here's an example of setting mobile metrics with the `Driver()` format:

```python
from seleniumbase import Driver

driver = Driver(mobile=True, d_width=360, d_height=640, d_p_r=2)
```
